lib/Model/Binary.cpp

```diff
@@ -157,6 +157,18 @@ bool Binary::verifyTypes(VerifyHelper &VH) const {
     auto Name = Type->name();
     if (not Names.insert(Name).second)
       return VH.fail(Twine("Multiple types with the following name: ") + Name);
+
+    if (const auto
+          *const T = llvm::dyn_cast<model::CABIFunctionType>(Type.get())) {
+      if (getArchitecture(T->ABI()) != Architecture())
+        return VH.fail("Function type architecture differs from the binary "
+                       "architecture");
+    } else if (const auto *const
+                 T = llvm::dyn_cast<model::RawFunctionType>(Type.get())) {
+      if (T->Architecture() != Architecture())
+        return VH.fail("Function type architecture differs from the binary "
+                       "architecture");
+    }
```

Comment:
`* const` -> `*`

Not much benefit but hinder readability.
One day `const` will be default and we'll have to say `mut`, but that day is not today.

Reply:
Done. As a habit I make all non-mutated variables const.